src/components/users/LoginForm/LoginForm.js

```diff
@@ -1,24 +1,54 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+
 import "style/common/_common.scss";
 import CommonInput from "components/common/CommonInput";
+import { userActions } from "actions/userActions";
+import { userActionTypesConstants } from "constants/userConstants";
 
 import "./login-form.scss";
 
-const LoginForm = () => (
-  <form className="login-form">
-    <CommonInput
-      label="Email"
-      inputType="email"
-      placeHolder="Type your email"
-    />
-    <CommonInput
-      label="Password"
-      inputType="password"
-      placeHolder="Type your password"
-    />
-    <button type="button" className="regular-text">
-      Log in
-    </button>
-  </form>
-);
+const { USER_REQUEST } = userActionTypesConstants;
+
+const LoginForm = () => {
+  const dispatch = useDispatch();
+
+  const [inputs, setInputs] = useState({ email: "", password: "" });
+  const { email, password } = inputs;
+
+  const handleChange = ({ target: { name, value } }) => {
+    setInputs((inputs) => ({ ...inputs, [name]: value }));
+  };
+
+  const handleSubmit = (e) => {
+    dispatch({ type: USER_REQUEST });
+    e.preventDefault();
+    dispatch(userActions.login(email, password));
+  };
+
+  return (
+    <form className="login-form" onSubmit={handleSubmit}>
+      <CommonInput
+        name="email"
+        label="Email"
+        inputType="email"
+        placeHolder="Type your email"
```

Comment:
It's a good practice using i18n in your texts across your site, even if you are supporting only one language at the moment your are developing your website. It's easier adding internationalization since the beginning than implementing it before and change all your texts inside the app

Reply:
Do I have to use useIntl?